sql/modules/Roles.sql

```diff
@@ -434,15 +565,26 @@ SELECT lsmb__grant_perms('contact_edit', 'eca_to_contact', 'ALL');
 SELECT lsmb__grant_perms('contact_edit', 'eca_to_location', 'ALL');
 SELECT lsmb__grant_perms('contact_edit', 'eca_tax', 'ALL');
 
-SELECT lsmb__create_role('contact_delete');
+SELECT lsmb__create_role('contact_delete',
+                         $DOC$
+                         This role allows editing of existing contacts.
+
+                         Combine this role with one or more 'contact_class_*' roles to be able to access
+                         contacts of the specific class. By itself, this role does not provide sufficient rights.
+                         $DOC$
+);
```

Comment:
does it to the same as `contact_edit` or does it allow users to delete contacts? and I'm guessing it doesn't do anything on its own?

Also, this role should probably be mentioned in the descriptions of the `contact_class_*` roles?

Reply:
Not the same as 'edit': with this role, you can search them, but not do anything with them but to delete them (so: no editing; no creation of new ones).

Reply:
ah, I was referring to the `This role allows editing of existing contacts.` which should have been something like `allows for removal...`. But I think this was fixed already?